### ISSUE TITLE
More verbose error message

### ISSUE THREAD

**Issue by Real-Gecko** [xtrinch/fcm-django#201]
I had a misconfigured APNS on FCM and was receiving `InvalidArgumentError('Request contains an invalid argument.')` from fcm-django while trying to send message to iOS, which was totally unclear to me. By using debugger I was able to find that exact response from FCM is
```json
{
    "error": {
        "code": 400,
        "message": "Request contains an invalid argument.",
        "status": "INVALID_ARGUMENT",
        "details": [
            {
                "@type": "type.googleapis.com/google.firebase.fcm.v1.FcmError",
                "errorCode": "INVALID_ARGUMENT"
            },
            {
                "@type": "type.googleapis.com/google.firebase.fcm.v1.ApnsError",
                "statusCode": 400,
                "reason": "DeviceTokenNotForTopic"
            }
        ]
    }
}
```
Which showed that problem is in APNS actually with certain error reason. Is it possible to get more verbose error output without using debugger?

**Comment by eliezerp3** [comment]
Did u ever get post v1 fcm-django to work for iOS?

**Comment by Real-Gecko** [comment]
Yes it's working now, there was some kind of configuration issue in Firebase related to APNS certificates and developer key, which I'm not aware of cause Firebase setup and iOS development is done by other people. However this reason `DeviceTokenNotForTopic` was a key to solving this, and `INVALID_ARGUMENT` was not clear to anyone. That's why I created this issue.

**Comment by eliezerp3** [comment]
Aight. I did all the setup steps and can’t figure out for the life of me why I’m not receiving the messages as the output has no errors. You are using the env variable pointing to path or passing the certificate path as a parameter to initialiaze_app?

**Comment by Real-Gecko** [comment]
By the way, to actually get notification on iOS(actual popup) you need to send notification like this:
```python
    message = Message(data=data, notification=Notification(title=title, body=body))
```
As Android developers said to me it's it's only sufficient to have `data` with `title` and `body` keys, but for iOS you need actual `Notification` instance added to the message.

**Comment by xtrinch** [comment]
@Real-Gecko Did the error messages not write itself onto the devices, visible in the admin panel?

**Comment by Real-Gecko** [comment]
Did not check that, because I don't use admin. But only message I got is mentioned in op.

**Comment by xtrinch** [comment]
We could write that error onto the device(s) for which it failed to send to, so that would solve this issue.
